lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -8,17 +8,15 @@ def sync
       @found = st.success?
 
       if @found
-        deployment_data = JSON.parse(raw_json)
-        @desired_replicas = deployment_data["spec"]["replicas"].to_i
-        @latest_rs = find_latest_rs(deployment_data)
-
-        @rollout_data = { "replicas" => 0 }.merge(deployment_data["status"]
+        @deployment_data = JSON.parse(raw_json)
```

Comment:
Its going to have to be an instance var if we want to use it in `minimum_unavailable_replicas_to_succeeded` (or whatever it gets called).

Reply:
> Well that or an instance var that just captures the value

Yeah, if it's only one more value that you need, I'd be explicit about capturing that (and potentially resetting it on `sync` cycles that err) rather than the whole blob.